src/tracepoint_format_parser.cpp

```diff
@@ -90,33 +90,19 @@ bool TracepointFormatParser::parse(ast::ASTContext &ctx, BPFtrace &bpftrace)
           if (format_file.fail()) {
             // Errno might get clobbered by LOG().
             int saved_errno = errno;
-
-            // Do not fail if trying to attach to multiple tracepoints
-            // (at least one of them could succeed)
-            bool fail = probe->attach_points.size() == 1;
```

Comment:
I assume that we will just print a warning here and fail elsewhere, is that correct?

Reply:
Yeah exactly. This should fail to attach and warn/error/ignore as per the config.